### PR TITLE
external.sh and add _with_tag targets for external tests

### PR DESCRIPTION
**Pull request by smlambert**
This PR is meant to serve as an example to add/enable xxxxxx_with_tag targets for all external tests.

Grinder: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2223/console
   [echo] Generate dockerfile!!! : build_image.sh derby openjdk8 hotspot ubi jdk full
     [exec] ERROR: Dockerfile generation for /dockerfile////Dockerfile.. failed.
     [exec] /Users/shelley_lambert@ca.ibm.com/work2/openjdk-tests/TKG/../../jvmtest/external/build_image.sh: line 17: ./common_functions.sh: No such file or directory
     [exec] /Users/shelley_lambert@ca.ibm.com/work2/openjdk-tests/TKG/../../jvmtest/external/build_image.sh: line 18: ./dockerfile_functions.sh: No such file or directory

Working out some issues finding/running the common_functions.sh / dockerfile_functions.sh files

Another issue, may need to set permissions differently 
(see output: mkdir: /dockerfile///: Read-only file system)

### CODE REVIEW

**Comment by smlambert** [comment]
FYI @james-crowley

**Comment by smlambert** [comment]
New grinder, shows I guess I better set a default ANT_VERSION

https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2229/console

File: external/derby/build.xml [hunk 1]
```diff
@@ -21,9 +31,30 @@
...
+			<fileset dir="${top}" includes="*.sh"/>
+		</copy>
+	</target>
+
+	<target name="build_dockerfile" depends="move_scripts" description="build derby test dockerfile">
+		<echo message="Generate dockerfile!!! : build_image.sh ${TEST} ${VERSION} ${VM} ${OS} ${PACKAGE} ${BUILD}" />						
+		<exec executable="bash">
+			<arg value="${EXT_DEST}/build_image.sh"/>
+			<arg value="${TEST}"/>
+			<arg value="${VERSION}"/>
```
Comment (james-crowley) [inline external/derby/build.xml]: This value needs to be a number. Like 8, 9, 10, etc. Yours was `openjdk11`.

**Comment by james-crowley** [comment]
@smlambert The version was off. See my comment above. It also appears the `build_image` step is building the old stock image, not the one built.

File: external/derby/build.xml [hunk 2]
```diff
@@ -21,9 +31,30 @@
...
+			<arg value="${VM}"/>
+			<arg value="${OS}"/>
+			<arg value="${PACKAGE}"/>
+			<arg value="${BUILD}"/>
+		</exec>
+	</target>
+
+	<target name="build_image" depends="build_dockerfile" description="build derby test docker image">
```
Comment (james-crowley) [inline external/derby/build.xml]: You can take this step completely out. The `build_image` script should generate the image and build said generated Dockerfile.

**Comment by smlambert** [comment]
Still have some clean up as the clean_image targets in ant are no longer valid, nor are the test targets in playlists that do not take DOCKER_IMAGE tags.  They can be removed and calling the '_with_tag' targets can be called with 'nightly' tag (which will then use the ubuntu base).

**Comment by smlambert** [comment]
Grinders:
camel (on ubi): https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2357
jenkins (on centos): https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2358
derby (on alpine): https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2359

**Comment by james-crowley** [comment]
@ShelleyLambert Looks like all the grinders are failing at building the images. Looks like the error is:

```
build_image:
00:46:22       [echo] Executing external.sh --build --dir jenkins --tag centos --version 11 --impl hotspot 
00:46:22       [exec] bash: /home/jenkins/workspace/Grinder/openjdk-tests/TKG/../../jvmtest/external/external.sh: No such file or directory
00:46:22       [exec] Result: 127
```

So it looks like `external.sh` is not being found, thus the images are not being built. Which leads to this error:

```
00:46:25  Unable to find image 'adoptopenjdk-jenkins-test:11-jdk-centos-hotspot-full' locally
```

**Comment by smlambert** [comment]
yep, its because I moved move_scripts targets up a level to external/build.xml and it likely needs to be corrected

**Comment by smlambert** [comment]
Rerun grinders:
camel (on ubi): https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2362
jenkins (on centos): https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2363
derby (on alpine): https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2364

**Comment by smlambert** [comment]
Closing this in favour of #1660

**Pull request closed without merging**